config/locales/views/wiki/show/en.yml

```diff
@@ -3,8 +3,8 @@
 en:
   wiki:
     show:
-     page_has_no_tags: "This page has no tags yet; help connect it to other content by <a onClick='$('#taginput').focus()'>adding descriptive tags below</a>."
-     revision_from: "This is a revision from %{time}."
+     page_has_no_tags: "This page has no tags yet; help connect it to other content by <a href='javascript: document.querySelector('.tag-input').focus();'>adding descriptive tags below</a>"
```

Comment:
The string seems to have single quotes nested inside single quotes.

Reply:
Yes, good catch! So, can the inner ones be escaped?

Reply:
Maybe we can use single quotes on the outer string, then double quotes on the JS string, and finally *escaped* single quotes on the innermost string like this: 
`'... <a href="javascript: document.querySelector(\'.tag-input\').focus();">adding ...'`
What do you think @jywarren?

Reply:
That sounds good. @g33kyshivam want to give this a try?